### ISSUE TITLE
[no-setup-in-describe] Symbol() considered problem

### ISSUE THREAD

**Issue by GradedJestRisk** [lo1tuma/eslint-plugin-mocha#295]
## Summary
I came against rule activation on calls on ECMAScript `Symbol` class.
I think it should not happen, as it does not affect the global state.

## Details
https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-setup-in-describe.md
> This rule reports all function calls and use of the dot operator (due to getters and setters) directly in describe blocks.

This code is rejected
``` javascript
const assert = require('assert/strict');
describe('no-setup-in-describe', () => {

  // ESLint: Unexpected function call in describe block.(mocha/no-setup-in-describe)
  const foo = Symbol('bar');

  it('foo should not be null', async () => {
    assert.notStrictEqual(foo, null);
  });

});
```

[Symbol](https://developer.mozilla.org/fr/docs/orphaned/Web/JavaScript/Reference/Global_Objects/Symbol) does merely instantiate a non-mutable object. 

Configuration:
- node 16.5
- eslint 7.27
- eslint-plugin-mocha 9.0

**Comment by lo1tuma** [comment]
Closing this as not planned.

`no-setup-in-describe` now documents a stricter default: direct calls in suite bodies are treated as setup code, and builtins are not special-cased. Projects that want to allow specific direct calls such as `Symbol()` can use the new rule-local `allow` option introduced in #424.
